_docs/services/intro.md

```diff
@@ -73,7 +73,7 @@ To list all the managed services and plans available to a given space, you run `
     <li><strong>Beta</strong>: The service is stable but still requires further development to ensure it can be deployed to production systems.</li>
     <li><strong>Alpha</strong>: The service is under development and some downtime or data loss can occur.</li>
     <li><strong>Deprecated</strong>: The service is deprecated and will be removed in a future release - please update.</li>
-
+    <li><strong>Legacy</strong>: The service is deprecated, and has been superseded by a preferred fully-supported service.<li>
```

Comment:
Since we're adding `Legacy` to distinguish from `Deprecated`, then In the table above this section, I think we need to update the `cdn-route` and `custom-domains` rows to appear as `Legacy`.

It looks like that data is controlled here: https://github.com/cloud-gov/cg-site/blob/main/_data/services.yml